.devcontainer/Dockerfile.devcontainer

```diff
@@ -46,6 +45,9 @@ RUN cp tinkey /usr/bin/
 RUN cp tinkey_deploy.jar /usr/bin/
 RUN rm tinkey tinkey_deploy.jar tinkey.bat $TINKEY_VERSION.tar.gz
 
+RUN pip install -r requirements.txt
+RUN pre-commit install
```

Comment:
Does this automatically add the hook to `git commit` in KMS's devcontainer?

Reply:
Yes, and anything else which uses the image built from this dockerfile

Reply:
Sounds good. How about adding it to the dev repo's devcontainer as well so that it's triggered to from that image/codespace?